co-authors-plus-social-pack.php

```diff
@@ -40,18 +40,20 @@ function __construct(){
 			add_action( 'admin_notices', array( $this, 'action_admin_notices_missing_coauthors_plus' ) );
 		}
 
-		add_filter( 'jetpack_sharing_twitter_via', 		array( $this, 'filter_jetpack_sharing_twitter_via' ), 		10, 2 );
-		add_filter( 'jetpack_sharing_twitter_related', 	array( $this, 'filter_jetpack_sharing_twitter_related' ), 	10, 2 );
+		add_filter( 'jetpack_sharing_twitter_via',				   array( $this, 'filter_jetpack_sharing_twitter_via' ),	 10, 2 );
+		add_filter( 'jetpack_sharing_twitter_related',			   array( $this, 'filter_jetpack_sharing_twitter_related' ), 10, 2 );
 
-		add_filter( 'coauthors_guest_author_fields', 	array( $this, 'filter_coauthors_guest_author_fields' ), 	10, 2 );
+		add_filter( 'coauthors_guest_author_fields',			   array( $this, 'filter_coauthors_guest_author_fields' ),	 10, 2 );
 
-		add_action( 'add_meta_boxes', 					array( $this, 'action_add_meta_boxes' ), 					20, 2 );
+		add_action( 'add_meta_boxes',							   array( $this, 'action_add_meta_boxes' ),					 20, 2 );
+
+		add_filter( 'coauthors_guest_author_personal_export_data', array( $this, 'filter_export_extra_data' ),				 10, 2 );
```

Comment:
Per the latest feedback, this needs a small update to `coauthors_guest_author_personal_export_extra_data` (https://github.com/Automattic/Co-Authors-Plus/pull/528#discussion_r194557539)